MetamodelAlgemeen.md

```diff
@@ -309,6 +309,12 @@ Voorbeelden van gegevens zijn de waardes ‘Jan’ en ‘1-1-1970’ betreffende
 Gegevens zijn de objectief waarneembare neerslag of registratie van feiten op een bepaald medium, zodanig dat deze gegevens uitgewisseld en voor langere tijd bewaard kunnen worden. Dat kan op papier, in digitale vorm, et cetera. Met deze gegevens wordt een model (een selectief deel dus) van de werkelijkheid vastgelegd in de tijd. Ofschoon de werkelijkheid nooit stilstaat, kan deze door het vastleggen van de gegevens toch worden bevroren. 
 Het waargenomen feit dat twee objecten een relatie hebben wordt ook gezien als een gegeven.
 
+Zodra een gegeven vastgelegd wordt of uitgewisseld wordt noemen we deze ook wel data. Het is deze data die waarde/data van een eigenschap van een object is, bv. 'Jan' is de waarde/data van de eigenschap voornaam van een Natuurlijk persoon. Data aangevuld met de betekenis uit het informatiemodel, waarin de definitie van wat een voornaam is staat, noemen we informatie.
+
+Er zijn een aantal modelelementen waar gegevens "in" zitten, oftewel waarvoor een feit waargenomen kan worden die wel als een waarde/data kunnen vastleggen. Dit zijn: _attribuutsoort_, _relatiesoort_ en _externe koppeling_. Elk gegeven kan:
```

Comment:
> Er zijn een aantal modelelementen waar gegevens "in" zitten [...]

Ik zou niet willen zeggen dat er gegevens in modelelementen zitten. Modelelementen worden gebruikt om de structuur/de onderdelen van gegevens te modelleren/voor te schrijven.

> [...] oftewel waarvoor een feit waargenomen kan worden die wel als een waarde/data kunnen vastleggen.

Gegevens hoeven niet:
- door waarneming tot stand te komen - bijv. een gegeven wat is afgeleid uit het toepassen van regels;
- niet over feiten te gaan - bijv. een bewering van een verdachte persoon die niet-feitelijk is;
- niet vastgelegd te zijn - bijv. een gegeven wat afgeleid kan worden, maar verder nergens is geëxpliciteerd.

> Dit zijn: _attribuutsoort_, _relatiesoort_ en _externe koppeling_.

Volgens mij bedoel je hier modelelementen die aan een kenmerksoort refereren en daarmee het belangrijkste onderdeel van de gegevensstructuur definiëren. In dat geval mist er volgens mij in dit lijstje nog de Relatieklasse. Die refereert namelijk ook aan een kenmerksoort.

Verder zijn er natuurlijk veel meer modelelementen die de structuur van gegevens beïnvloeden.

Reply:
Ik zou het als volgt willen formuleren:

Er zijn een aantal modelelementen die "gegevens kunnen bevatten", ofwel waarvoor een feit waargenomen kan worden dat we kunnen vastleggen als de data/waarde van dat modelelement.

@pano Dat geldt volgens mij niet voor een relatieklasse, maar voor de attribuutsoorten daarvan.

Reply:
> Er zijn een aantal modelelementen die "gegevens kunnen bevatten", ofwel waarvoor een feit waargenomen kan worden dat we kunnen vastleggen als de data/waarde van dat modelelement.

Hier kan ik een groot deel van mijn commentaar voor herhalen. Aanvullend:
Data is voor mij niet synoniem aan waarde. Data is voor mij de concrete vastlegging van gegevens op een bepaald medium.
Een modelelement kan overigens geen waarde hebben. Een gegeven wel.

Ik zou het formuleren als: Er zijn een aantal soorten modelelementen die het kenmerk van een corresponderend gegeven bepalen en daarmee vormend zijn voor de gegevens die corresponderen met dat modelelement. 

(vormend in de zin dat ze naast kenmerksoort, ook objecttype en waardetype bepalen)

> @Pano Dat geldt volgens mij niet voor een relatieklasse, maar voor de attribuutsoorten daarvan.

Een relatieklasse modelleert wel degelijk een gegevenstype, namelijk een relatiegegeven tussen twee objecten. Dit relatiegegeven wil je als onderwerp van andere gegevens gebruiken, daarom gebruik je een relatieklasse. Maar het relatiegegeven blijft bestaan.